doc/api/debugger.md

```diff
@@ -202,3 +202,23 @@ To start debugging, open the following URL in Chrome:
 ```
 
 [TCP-based protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+
+## Trace Event Support for Node.js
```

Comment:
Does this belong in this file ?  I see the debugger as being 2 separate things and therefore trace should be in its own file doc/api/tracing.md.